overlord/devicestate/handlers_install.go

```diff
@@ -90,6 +91,28 @@ func writeModel(model *asserts.Model, where string) error {
 	return asserts.NewEncoder(f).Encode(model)
 }
 
+func writeLogs(rootdir string) error {
+	// XXX: would be great to use native journal format but it's tied
+	//      to machine-id, we could journal -o export but there
+	//      is no systemd-journal-remote on core{,18,20}
+	//
+	// XXX: or only log if persistent journal is enabled?
+	logPath := filepath.Join(rootdir, "var/log/install-mode.log")
+	if err := os.MkdirAll(filepath.Dir(logPath), 0755); err != nil {
+		return err
+	}
+
+	f, err := os.Create(logPath)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+
+	cmd := exec.Command("journalctl")
```

Comment:
there should always be only the log from current but, but maybe as extra safety we should call `journalctl -b 0`.